lib/domain.gi

```diff
@@ -389,3 +401,4 @@ InstallGlobalFunction( InstallAccessToGenerators,
         end );
 
     end );
+
```

Comment:
Why add this line?
```suggestion
```

Reply:
Was done by accident. I've removed it now.